### ISSUE TITLE
Small caps fails in Noto Sans Regular but works in Noto Sans Display

### ISSUE THREAD

**Issue by StatisticalProgrammer** [googlefonts/noto-fonts#2028]
### Font 

NotoSans-Regular.ttf  
File version 1.04

###  Site: [https://github.com/googlefonts/noto-fonts/tree/main/hinted/ttf/NotoSans](https://github.com/googlefonts/noto-fonts/tree/main/hinted/ttf/NotoSans)
  >   Date: ‎2021-03-11

### Font Version
  > * Win -- Wednesday, ‎27 ‎February, ‎2013, ‏‎12:00:00 AM

### Application name and version
  > Moot: TeXworks and TeXstudio

### Issue
Small caps fail on TTF. (They work with OTF)
  1. Replication: Reinstalled on 2 separate Windows 10 systems, 2+ times each
  2. Observation: Small caps fail for the regular Sans, but not for display sans
  3. Expectation: Small caps should work for regular Sans

The problem was solved by using the hinted OTF version, which hidden in a tree in which the first branch is erroneously labeled "unhinted". 
The final branch of the tree is labeled "hinted and unhinted", which corrects the earlier mislabeling ... in a way.

**Comment by StatisticalProgrammer** [comment]
Noto Sans Regular does not have functioning **small caps,** while the other styles do, according to my LaTeX use (FontSpec; LuaTeX, ProTeXt with MikTeX and TeXStudio) on Windows 10. 

Noto Sans Display Regular does have functioning small caps.

**Comment by moyogo** [comment]
@StatisticalProgrammer Noto Sans Regular has small caps for many Latin glyphs, none for Greek or Cyrillic glyphs, since version 1.902.

What version of Noto Sans Regular are you testing
How are you testing it?

I cannot reproduce with Fontspec and XeLaTeX:
```latex
\documentclass[a4paper]{l3doc}
\usepackage{fontspec}
\begin{document}
\fontspec{NotoSans-Regular.ttf}
abcdefghijklm

nopqrstuvwxyz

àáâãäåæçèéêëì

íîïðñòóôõöøùú

ûüýþÿāăąćĉċčď

đēĕėęěĝğġģĥħĩ

īĭįĳĵķĺļľŀłńņ

ňŋōŏőœŕŗřśŝşš

ţťŧũūŭůűųŵŷźż

žǻǽǿșțẁẃẅỳ

\fontspec{NotoSans-Regular.ttf}[Letters=SmallCaps]
abcdefghijklm

nopqrstuvwxyz

àáâãäåæçèéêëì

íîïðñòóôõöøùú

ûüýþÿāăąćĉċčď

đēĕėęěĝğġģĥħĩ

īĭįĳĵķĺļľŀłńņ

ňŋōŏőœŕŗřśŝşš

ţťŧũūŭůűųŵŷźż

žǻǽǿșțẁẃẅỳ

\end{document}
```
<img width="215" alt="image" src="https://user-images.githubusercontent.com/1923747/110783140-f2af2280-825f-11eb-96c2-db1d4a4ccd97.png">

**Comment by StatisticalProgrammer** [comment]
Thanks for your quick response. I shall edit the above response with more information.

I was using TeXStudio, with which I replicated the failure on another Windows 10 system a few weeks ago, until now. 

I used TeXWorks on your example (thanks!) and the error persists, when using Lua. (The default XeLaTeX does not work with fontspec, for me and I believe for everybody, as of today.) 

However, I did reinstall the .ttf fonts for Sans-Regular and for SansDisplay-Regular today, and then I restarted my system. The error persists. 

[SMallCapsXeLaTeX.pdf](https://github.com/googlefonts/noto-fonts/files/6123073/SMallCapsXeLaTeX.pdf)

FYI, I displayed Noto small caps when discussing Adobe Source Sans 3 [https://github.com/adobe-fonts/source-sans/issues/190#issuecomment-778039616](https://github.com/adobe-fonts/source-sans/issues/190#issuecomment-778039616)
